engine/src/flutter/lib/web_ui/lib/src/engine/platform_dispatcher/app_lifecycle_state.dart

```diff
@@ -104,7 +104,7 @@ class _BrowserAppLifecycleState extends AppLifecycleState {
     }
   });
 
-  void _onViewCountChanged(_) {
+  void _onViewCountChanged(int _) {
```

Comment:
(Slightly surprising that we need to specify the type here, but not in the lambdas!)

Reply:
There has been some discussion around this. The verdict was that top-level things will always need types (that's this one). Lambdas are not top-level and therefore don't have this enforced. https://dart.dev/tools/linter-rules/strict_top_level_inference

Reply:
ACK! Thanks for the link!